.husky/pre-commit

```diff
@@ -2,5 +2,3 @@
 . "$(dirname "$0")/_/husky.sh"
 
 yarn lint-staged
-
-yarn test
```

Comment:
@mdjmoore I removed running tests with pre-commit, since we run tests in PR already, so we can save the cost of running tests when people commit frequently 🙆‍♂️ Hope that sounds good to you too! Thank you!